common/constants.c

```diff
@@ -636,6 +636,15 @@ const p11_constant p11_constant_mechanisms[] = {
 	{ CKA_INVALID },
 };
 
+const p11_constant p11_constant_profiles[] = {
+	CT (CKP_BASELINE_PROVIDER, "baseline_provider")
```

Comment:
Let's use "kebab-case", rather than "snail_case" for consistency with other constants.